pkg/controller/utils/datadog/forwarders_manager.go

```diff
@@ -98,6 +100,17 @@ func (f *ForwardersManager) ProcessEvent(obj MonitoredObject, event Event) {
 	forwarder.eventChan <- event
 }
 
+// MetricsForwarderStatusForObj used to retrieve the Metrics forwarder status for a give object
+func (f *ForwardersManager) MetricsForwarderStatusForObj(obj MonitoredObject) *datadoghqv1alpha1.DatadogAgentCondition {
+	id := getObjID(obj)
+	forwarder, err := f.getForwarder(id)
+	if err != nil {
+		// forwarder not present yet
+		return nil
```

Comment:
shouldn't we at least log something here since we're ignoring the error ?

Reply:
I will add a log in the method caller